identity/client/src/pages/groups/modals/EditModal.tsx

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+
+import { FC, useEffect, useState } from "react";
+import { EditFormModal, UseEntityModalProps } from "src/components/modal";
+import useTranslate from "src/utility/localization";
+import { useApiCall } from "src/utility/api/hooks";
+import { useNotifications } from "src/components/notifications";
+import { Group, updateGroup } from "src/utility/api/groups";
+import TextField from "src/components/form/TextField";
+
+const EditModal: FC<UseEntityModalProps<Group>> = ({
+  entity: group,
```

Comment:
Optional: Since `group` can not be null, we can now use destructuring here. That's not required tho.

```
  entity: { id, name: currentName },
```

and then `currentName` can be used in
```
 const [name, setName] = useState(currentName);
```

and `id` can be used in the api call

```
      const { success } = await callUpdateGroup({
        id,
        name: name.trim(),
      });
```